RowField.h

```diff
@@ -44,6 +44,8 @@ typedef enum ReservedFields_ {
 
    /* Platform specific fields, defined in ${platform}/ProcessField.h */
    PLATFORM_PROCESS_FIELDS
+   PERCENT_CPU_GROUP = 130,
+   PERCENT_MEM_GROUP = 131,
```

Comment:
Global fields should be defined in the enum definition before platform-specific ones.

Reply:
I didn't see any place in htop codebase mentioning that global fields should be before platform specific ones. Personally I think htop has no specific rules for assigning field IDs and the current allocation is messy. Ideally the fields IDs should be stable across htop releases so the configuration files can refer the fields by numbers. In addition, we should reserve ranges of field IDs for specific purposes so that, e.g. dynamic fields would not use them causing collisions.

I would cite [MediaWiki namespace IDs](https://mediawiki.org/wiki/Extension_default_namespaces) as a good example of how IDs can be allocated are how IDs are reserved for future uses.

Reply:
It's not a matter of numeric values; just a code ordering. The macro including the platform flags should be the last line in that definition.

Regarding actual allocations: Old versions of htop used to write the field IDs into the config files. With more recent versions this has been changed to use name representations instead, thus the only thing that needs to be kept for BC is with the columns available with old htop versions that still stored the numeric IDs.

But yes, the current assignment is kinda messy, but nothing too easy to fix over night.